code/game/turfs/simulated/floor.dm

```diff
@@ -253,3 +253,6 @@
 			G.anchored = TRUE
 			return TRUE
 	return FALSE
+
+/turf/open/floor/get_dumping_location()
```

Comment:
Actually should probably be /open, don't see a reason why you shouldn't be able to dump in space.